lib/active_job/queue_adapters/advanced_sneakers_adapter.rb

```
@@ -29,12 +29,29 @@ def enqueue_at(job, timestamp) #:nodoc:
+          strategy = strategy_value.respond_to?(:call) ? strategy_value.call : strategy_value
+
+          case strategy.to_sym
+          when :leveled
+            AdvancedSneakersActiveJob.leveled_delayed_publisher
+          when :legacy
+            AdvancedSneakersActiveJob.delayed_publisher
+          else
+            ::ActiveJob::Base.logger.warn { "AdvancedSneakersAdapter: unknown delayed_delivery #{strategy.inspect}, using :legacy" }
+            AdvancedSneakersActiveJob.delayed_publisher
+          end
+        end
```

RuboCop (1.87.0)

[convention] Method has too many lines. [11/10]

(Metrics/MethodLength)